project-management/FujitsuMeetingDates

```diff
@@ -0,0 +1,23 @@
+# Fujitsu/Turing Sprint Meetings
+## Meetings Descriptions
+Each sprint is 9 days (approx. 2 work weeks long) therefore each of these meetings occurs every 2 weeks. 
+The whole Alan Turing and Fujitsu team are invited to these meetings and a member of the community will be included in retrospective meetings.
+*Backlog refinement* is carried out before each sprint to reprioritise the backlog of user stories (issues) ahead of the next sprint planning meeting.
+*Sprint Planning* is carried out at the start of each sprint and is when the user stories (issues) that are planned to be completed in that sprint are chosen.
+*Sprint retrospective* is carried out at the end of each sprint to consider how the sprint went and any changes that need to be made to the process before the next sprint.
+During this meeting a demo of the achievements of the previous sprint will be shown. 
+
+More details about the Agile process can be found in the Agile - Open source workflow document also in the project management folder. 
+https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/EllenDevereux-patch-2/project-management/agile-opensource-workflow.md
+
+### Meetings:
+
+|Dates |Time| Meeting Description|
+|---|---|----------|
+| 27/03|14:00-15:00| Demo for sprint 1|
+| 31/03|14:00-15:00| Backlog Refinement for Sprint 2|
+| 01/04|10:00-11:00| Retrospective for Sprint 1 |
+| 02/04|10:00-11:00| Sprint planning for Sprint 2 |
```

Comment:
We've missed this demo date - 27 March - I think.

I can't make the meeting on 31 March, I might be able to make the meetings on 1 & 2 April but I won't attend if I haven't seen the demo & sprint report in advance.

Reply:
This was discussed in todays meeting. (Notes of which are being uploaded by Georgia) I think it was agreed that since the user stories require more clarification and more documentation is required before building can go ahead then there's no need for a video (there's no interactive platform to describe) a sprint report is being released still by Chris Legg.